packages/web/src/tools/_utilities.scss

```diff
@@ -0,0 +1,27 @@
+@use 'sass:list';
```

Comment:
Please add a credit to Bootstrap here and in the calling file, we want to be the nice guys 🙂.

Reply:
It is rather inspired by RUI, but I will mention them both.